pkg/webui/console/components/gateway-data-form/index.js

```diff
@@ -141,10 +164,27 @@ class GatewayDataForm extends React.Component {
     this.setState({ shouldDisplayWarning: this.isNotValidDuration(value) })
   }
 
+  @bind
+  setFrequencyPlanTouched() {
+    const { setFieldTouched } = this.props.formRef.current
+    setFieldTouched('frequency_plan_id')
```

Comment:
Actually, I think we can mark select fields touched by default, when a value was selected:

```diff
diff --git a/pkg/webui/components/select/index.js b/pkg/webui/components/select/index.js
index 66822b752..b99fe50bb 100644
--- a/pkg/webui/components/select/index.js
+++ b/pkg/webui/components/select/index.js
@@ -105,7 +105,7 @@ class Select extends React.PureComponent {
       await this.setState({ value })
     }
 
-    onChange(value)
+    onChange(value, true)
   }
 
   @bind
```